src/main/java/org/broadinstitute/hellbender/tools/spark/pipelines/ReadsPipelineSpark.java

```diff
@@ -63,9 +64,22 @@
     @Argument(doc = "If specified, shard the output bam", shortName = "shardedOutput", fullName = "shardedOutput", optional = true)
     private boolean shardedOutput = false;
 
-    @Argument(doc = "the join strategy for reference bases", shortName = "joinStrategy", fullName = "joinStrategy", optional = true)
+    @Argument(doc = "the join strategy for reference bases and known variants", shortName = "joinStrategy", fullName = "joinStrategy", optional = true)
     private JoinStrategy joinStrategy = JoinStrategy.SHUFFLE;
-    
+
+    /**
+     * all the command line arguments for BQSR and its covariates
+     */
+    @ArgumentCollection(doc = "all the command line arguments for BQSR and its covariates")
+    private final RecalibrationArgumentCollection bqsrArgs = new RecalibrationArgumentCollection();
+
+    /**
+     * command-line arguments to fine tune the apply BQSR step.
+     */
+    @ArgumentCollection
+    public ApplyBQSRArgumentCollection applyBqsrArgs = new ApplyBQSRArgumentCollection();
```

Comment:
Shouldn't this be a `ApplyBQSRUniqueArgumentCollection`, with a conversion `toApplyBQSRArgumentCollection()` below?